src/rest/crypto/lastTrade.ts

```diff
@@ -0,0 +1,23 @@
+// CF: https://polygon.io/docs/crypto/get_v1_last_crypto__from___to
+
+import { get } from "../transport/request";
+
+export interface ICryptoLastTrade {
+  status?: string;
+	request_id?: string;
+  symbol?: string;
+  last?: {
+		conditions?: number[];
+		exchange?: number;
+		price?: number;
+		size?: number;
+		timestamp?: number;
+	}
```

Comment:
Some of these aren't optional, correct?

I'm seeing quite a few interfaces defined with optional properties. Are there cases in which the response doesn't have any of these properties?

Reply:
The spec has most response attributes listed as optional. Its pretty rare to have a required one.

And from how the spec reads, it looks like a completely empty response is a possibility